ItemPool.py

```diff
@@ -10,7 +10,7 @@
 
 
 #This file sets the item pools for various modes. Timed modes and triforce hunt are enforced first, and then extra items are specified per mode to fill in the remaining space.
-#Some basic items that various modes require are placed here, including pendants and crystals. Medallion requirements for the two relevant entrances are also decided.
+#Some basic items that various modes require are placed here, including stones and medallions. 
```

Comment:
Changing this comment isn't allowed D:

In all seriousness tho, this comment is completely wrong and is just a leftover from alttpr, so if we're going to touch it, it should just be removed entirely imo. (Aside from maybe setting the item pools, but that's pretty obvious from the name of the file and functions)